pytorch_blade/src/torch-mlir/lib/Conversion/MhloPasses.cpp

```diff
@@ -55,6 +55,7 @@ void mlir::torch::createDiscTorchBackendToMhloBackendPipeline(
   pm.addNestedPass<func::FuncOp>(createDiscConvertTorchToMhloPass());
   pm.addNestedPass<func::FuncOp>(createConvertTorchToMhloPass(
       /*enableStaticShape*/ false, /*enableI32Index*/ true));
+  pm.addNestedPass<func::FuncOp>(createDiscLowerOperatorOpPass());
```

Comment:
confused with the pass name, `Op` is also short for the operator, this pass converts PyTorch custom op into mhlo, and maybe we do not need an independent pass to deal with the custom op, how about renaming it to `DiscConvertTorchToDiscMhlo`, that we can also add other op conversions that lower to disc-mhlo in this pass in the future.

Reply:
> confused with the pass name, Op is also short for the operator ...

As @tanyokwok referenced, there is an [OperatorOp](https://github.com/llvm/torch-mlir/blob/main/include/torch-mlir/Dialect/Torch/IR/TorchOps.td#L832) ...

> maybe we do not need an independent pass to deal with the custom op ...

From a perspective of functionality, `DiscLowerOperatorOpPass` should be part of `DiscConvertTorchToMhloPass`. But as `mhlo_disc::MhloDiscDialect` is not among legal output dialects of `ConvertTorchToMhloPass`, which is from torch-mlir project, our conversion pattern has to be put into another pass and placed after `ConvertTorchToMhloPass`. A radical solution may be to make `ConvertTorchToMhloPass` open to dialect registry.

Reply:
Our `disc_mhlo` op is not part of `MHLO`, so to put those op conversions into `ConvertTorchToMhloPass` doesn't make sense. I think the naming `DiscConvertTorchToDiscMhlo` is much more intuitive.

Reply:
The another PR https://github.com/alibaba/BladeDISC/pull/634 added a pass that convert mhlo::rng to disc-mhlo::rng in `tao_compiler_main`, if we add `DiscConvertTorchToDiscMhlo ` here, maybe we can implement all disc-mhlo conversions in this pass.

Reply:
make sense .. changed to `DiscConvertTorchToDiscMhlo`.